ncmcm/visualisers/latent_space.py

```diff
@@ -123,11 +123,9 @@ def _plot_ps(self, fig, ax, colors=None, **kwargs):
 
         for i in range(len(self.y) - 1):
             d = (self.y[i + 1] - self.y[i])
-            kwargs.setdefault('arrow_length_ratio', 0.1 / np.linalg.norm(d))
-            kwargs.setdefault('linewidths', 1)
             ax.quiver(self.y[i, 0], self.y[i, 1], self.y[i, 2],
                       d[0], d[1], d[2],
-                      color=color_dict[self.b[i]], **kwargs)
+                      color=color_dict[self.b[i]], arrow_length_ratio=0.1/np.linalg.norm(d), linewidths=1, **kwargs)
```

Comment:
Note that this reverts a change made in PR #20, because at least on my side the current version made the plot_phase_space confusing/messy (both when using PyTorch and TensorFlow) because of the arrows.

Example:
![phase_space](https://github.com/NC-MCM/NC-MCM/assets/38300176/6cdc2f5a-e3ca-4b74-a9e8-cd264a135936)